tox.ini

```diff
@@ -12,9 +12,9 @@ deps =
     beautifulsoup4==4.3.2
     mock==1.0.1
     pytest-cache==1.0
-    pytest==2.7.2
-    pytest-cov==1.8.1
-    pytest-django==2.8.0
+    pytest==2.8.3
+    pytest-cov==2.2.0
+    pytest-django==2.9.1
```

Comment:
There was a version conflict with the py library while installing the older version of pytest (since Travis comes with a version of pytest preinstalled), hence the upgrade. Should check whether the explicit install of the same version in `.travis.yml` is still needed though...